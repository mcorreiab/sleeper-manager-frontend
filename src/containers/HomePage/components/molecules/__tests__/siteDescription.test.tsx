import renderer from "react-test-renderer";
import SiteDescription from "../siteDescription";

it("should render site description with title and description", () => {
  const tree = renderer.create(<SiteDescription />).toJSON();
  expect(tree).toMatchSnapshot();
});
