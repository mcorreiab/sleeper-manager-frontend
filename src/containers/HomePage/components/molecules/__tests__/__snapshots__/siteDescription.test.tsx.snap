// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render site description with title and description 1`] = `
<div>
  <h2
    className="mb-4 font-normal text-[2.4375rem] md:text-[2.85rem]"
  >
    Tool to easy your life at 
    <span
      className="text-[#f26506]"
    >
      sleeper
    </span>
     
    fantasy platform
  </h2>
  <p
    className="text-xl md:text-[1.4rem] font-['Open_Sans,sans-serif']"
  >
    Insert your username to check all players with non ok injury status
  </p>
</div>
`;
